src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int leg;
+    private int base;
+
+    public IsoscelesTrapezoid(String color, String name) {
+        setColor(color);
+        setName(name);
+    }
+
+    public int getBase() {
+        return base;
+    }
+
+    public void setBase(int base) {
+        this.base = base;
+    }
+
+    public int getLeg() {
+        return leg;
+    }
+
+    public void setLeg(int leg) {
+        this.leg = leg;
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();

```

Comment:
This method is just calling the superclass method without any specific implementation for IsoscelesTrapezoid. You should implement the logic to calculate the area for an isosceles trapezoid here.